src/contexts/CurrentUserContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
```

Comment:
in `SearchForm`

```jsx


    <form className='search-form' onSubmit={handleSubmit}>
      <input
        className='search-form__input'
        name='keyword'
        type='text'
        placeholder='Enter topic'
        onChange={handleChange}
        required
      />
```


If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components